kolibri/plugins/learn/assets/src/views/AlsoInThis.vue

```diff
@@ -24,31 +24,34 @@
           :kind="content.learning_activities"
         />
         <KIcon v-else class="topic-icon" icon="topic" />
-        <div class="content-meta" :style="{ top: (content.duration ? '-8px' : '8px') }">
-          <TextTruncator
-            :text="content.title"
-            :maxHeight="24"
-            :style="{ marginTop: (content.duration ? '8px' : '0px') }"
-          />
-          <TimeDuration
-            v-if="content.duration"
-            :style="{
-              color: $themeTokens.annotation,
-              top: '16px',
-              position: 'relative'
-            }"
-            :seconds="content.duration"
-          />
-        </div>
-        <div class="progress">
-          <KIcon
-            v-if="content.progress === 1"
-            icon="star"
-            class="mastered-icon"
-            :style="{ fill: $themeTokens.mastered }"
-          />
-          <ProgressBar v-else :contentNode="content" class="bar" />
+
+        <div class="content-meta">
+          <div class="text-and-time">
+            <TextTruncator
+              class="content-title"
+              :text="content.title"
+              :maxHeight="72"
+            />
+            <TimeDuration
+              v-if="content.duration"
+              class="time-duration"
+              :style="{
+                color: $themeTokens.annotation,
+              }"
+              :seconds="content.duration"
+            />
+          </div>
+          <div class="progress">
+            <KIcon
+              v-if="withProgress(content).progress === 1"
```

Comment:
The progress bar already contains this conditional logic: https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/plugins/learn/assets/src/views/ProgressBar.vue#L11

And accesses progress via the contentNodeMap from the useContentNodeProgress composable - so no need to mix it into the content node first.

Can also remove this addition from the previous PR: https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/plugins/learn/assets/src/views/ProgressBar.vue#L60

Reply:
Thanks for the reminder - once I got the bits hooked up elsewhere and they worked I was too pleased to remember I had to make proper use of ProgressBar